query/src/main/scala/hmda/query/model/institutions/InstitutionQuery.scala

```diff
@@ -2,10 +2,30 @@ package hmda.query.model.institutions
 
 case class InstitutionQuery(
   id: String,
-  name: String,
-  cra: Boolean,
   agency: Int,
+  filingPeriod: Int,
+  activityYear: Int,
+  respondentId: String,
   institutionType: String,
-  parent: Boolean,
-  filingPeriod: Int
+  cra: Boolean,
+  emailDomain2015: String,
```

Comment:
Sure. I think removing the year labels on those fields would make them more accurate and descriptive.